src/ProductHub.Api/Program.cs

```diff
@@ -21,3 +21,5 @@
 app.MapControllers();
 
 app.Run();
+
+public partial class Program { }
```

Comment:
What is this?

Reply:
This is used in the WebApplicationFactory in the integration test. I am referring Microsoft's official document for integration for this.